api/build.gradle

```diff
@@ -243,7 +249,10 @@ dependencies {
 
 swaggerSources {
   workbench {
-    inputFile = file("$projectDir/$swaggerSourceFile")
```

Comment:
We still need to validate workbench.yaml, right?

Reply:
We're validating the merged file. Unless we define things in two places, workbench.yaml by itself won't be valid, as it will have references to things defined in client_api.yaml.